### PR TITLE
Add context menu for PluginsManager 

### PR DESCRIPTION
**Pull request by jjw24**
image

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs [hunk 1]
```diff
@@ -19,11 +20,60 @@ public ContextMenu(PluginInitContext context, Settings settings)
...
+            return new List<Result>
+            {
+                new Result
+                {
+                    Title = Context.API.GetTranslation("plugin_pluginsmanager_plugin_contextmenu_openwebsite_title"),
+                    SubTitle = Context.API.GetTranslation("plugin_pluginsmanager_plugin_contextmenu_openwebsite_subtitle"),
+                    IcoPath = "Images\\website.png",
+                    Action = _ => 
+                    {
+                        SharedCommands.SearchWeb.NewBrowserWindow(pluginManifestInfo.Website);
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs:34]: Why not use NewBrowserInTab?

File: Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs [hunk 2]
```diff
@@ -19,11 +20,60 @@ public ContextMenu(PluginInitContext context, Settings settings)
...
+                    }
+                },
+                new Result
+                {
+                    Title = Context.API.GetTranslation("plugin_pluginsmanager_plugin_contextmenu_gotosourcecode_title"),
+                    SubTitle = Context.API.GetTranslation("plugin_pluginsmanager_plugin_contextmenu_gotosourcecode_subtitle"),
+                    IcoPath = "Images\\sourcecode.png",
+                    Action = _ =>
+                    {
+                        SharedCommands.SearchWeb.NewBrowserWindow(pluginManifestInfo.UrlSourceCode);
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs:45]: Same as last one

**Review by taooceros — APPROVED** [review]

File: Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs (hunk 2 above)
Comment (JohnTheGr8) [reply Plugins/Flow.Launcher.Plugin.PluginsManager/ContextMenu.cs:45]: agreed, let's be consistent with this.

**Comment by jjw24** [comment]
Good to go? @JohnTheGr8 @taooceros

**Review by taooceros — APPROVED** [review]
Good to go!

**Pull request merged**